modular_stonehedge/code/datums/traits/unspecial.dm

```diff
@@ -547,7 +548,7 @@
 /datum/quirk/nymphomaniac
 	name = "Nymphomaniac"
 	desc = "I am so senstiive even my pants are enough to constantly arouse me... I must not wear pants or wear things that don't touch my groin. Atleast my experiences with certain things make me good in bed."
-	value = -3
+	value = -5 // this thing really really sucks to have. I watched someone suffer all round with it.
```

Comment:
I made it so it only happens if you wear pants, skirts or fishnets etc shouldnt trigger it anymore.